### PR TITLE
Various Fix to allow Placeholder Respawn

### PR DESCRIPTION
**Pull request by Wiggo32**
This will make sure all PH are able to respawn and fix any issues of 'Lost' mobs. I only changed the ones that followed the simple formula for handling PH's. Any NM's that had more complicated coding or only 1 Placeholder; I did not touch. They can be taken care of on a case by case basis.

### CODE REVIEW

**Comment by ymkc** [comment]
Problem with this might be if the server crashes after setting the PH, but before killing the NM, the db entry will still be there.

so there might be a (multi gameserver aware) cleanup job needed, to remove all the PHs on startup

File: scripts/zones/Palborough_Mines/mobs/Copper_Beetle.lua [hunk 1]
```diff
@@ -22,13 +22,15 @@ function onMobDespawn(mob)
         local checkPH = GetServerVariable("[PH]Bu_Ghi_Howlblade");
         if (ToD <= os.time() and GetMobByID(Bu_Ghi_Howlblade):isDead()) then
             -- Set ToD, DisallowRespawn PH repop, set NM respawn time
-            if (math.random(1,10) == 5 and checkPH == 0) then
```
Comment (TeoTwawki) [inline scripts/zones/Palborough_Mines/mobs/Copper_Beetle.lua]: no point in repeating the same commands once the already happened I tested those beetles and the nm last night there is no problem with having the var checked at the same time as the random number, being together doesn't make it "not used"

**Comment by TeoTwawki** [comment]
@ymkc @Wiggo32 
we could possibly switch to using local vars instead of server vars. local vars get auto-wiped on spawn. Would need to discuss on discord.

**Comment by TeoTwawki** [comment]
Wish I'd thpought of that when he showed us the pastebin on discord. 

Another way to clear them would be check the var on spawn and if mobID matched the var's PH ID wipe it. since that'd obv mean ph already popped and its not a deter'd/dissallowed pop due to server reboot. Or do what retail does and startup with NM windows "already open" and a new PH selected so they clear it. (maybe we already do, I'd have to look at every NM's default value sin the SQL+zone scripts..)

**Comment by TeoTwawki** [comment]
Resolution on discord:
localVar and a new listener to replace servervars, death to servervars. Torches, pitchforks, and partially spoiled produce will be handed out in a commit coming soon.

**Comment by Wiggo32** [comment]
Closing... moving on to a better solution.

**Pull request closed without merging**